fastlane/Fastfile

```diff
@@ -518,7 +518,7 @@ platform :ios do
               # and push the changes
               docs_destination_folder = "docs/#{version_number}"
               index_destination_path = "docs/index.html"
-              FileUtils.mv docs_generation_folder, docs_destination_folder
+              FileUtils.cp_r docs_generation_folder, docs_destination_folder
```

Comment:
this is the fix - the folder would get _moved_, so then when ruby goes to remove it from the tmp directory, it's not there anymore. 
Fixed by just doing a copy instead of a move. The copy will also get automatically cleaned up afterwards, since both the original and the copy are stored in temporary directories.